homeassistant/components/smartthings/sensor.py

```diff
@@ -346,6 +476,13 @@ def unit_of_measurement(self):
         unit = self._device.status.attributes[self._attribute].unit
         return UNITS.get(unit, unit) if unit else self._default_unit
 
+    @property
+    def last_reset(self) -> datetime | None:
+        """Return the time when the sensor was last reset, if any."""
+        if self._attribute in [Attribute.power, Attribute.energy]:
```

Comment:
Only energy has a last reset as it's a metered entity (always increasing value). Power is the current throughput.

```suggestion
        if self._attribute == Attribute.energy:
```

Reply:
yeah, as usual you are fully right 😄